Dockerfile.test

```diff
@@ -0,0 +1,17 @@
+FROM python:3.8.1-buster
```

Comment:
This should probably match the python version in `Dockerfile`. Also, I think we can use a simpler version tag like `python:3.9`

Reply:
3261d4d88bcebf40fbd0f440b55c7122a1dd8804